cli/cmd/top.go

```diff
@@ -506,9 +516,22 @@ func renderTable(table *topTable, requestCh <-chan topRequest, done <-chan struc
 			table.insert(req)
 		case <-ticker.C:
 			termbox.Clear(termbox.ColorDefault, termbox.ColorDefault)
+			width, _ = termbox.Size()
 			table.adjustColumnWidths()
-			table.renderHeaders()
-			table.renderBody()
+			tablewidth = table.tableWidthCalc()
+			table.renderHeaders(scrollpos)
+			table.renderBody(scrollpos)
+			termbox.Flush()
+		case offset := <-horizontalScroll:
+			if (offset > 0 && scrollpos < 0) || (offset < 0 && scrollpos > (width-tablewidth)) {
+				scrollpos = scrollpos + offset
+			}
+			termbox.Clear(termbox.ColorDefault, termbox.ColorDefault)
+			width, _ = termbox.Size()
```

Comment:
I could be wrong here but removing this line, did not break anything. I don't see why the scroll event should re-calculate the table width? WDYT?

The ticker event calculating makes sense, as new data is added, the width might change.

Reply:
You're right, All the lines written in case `ticker.c` need not be repeated again in the case `horizontalScroll`. I have removed the same, it did not break any functionality.